addons/api/mirage/generated/factories/host-catalog.js

```diff
@@ -11,4 +11,13 @@ export default Factory.extend({
   created_time: () => date.recent(),
   updated_time: () => date.recent(),
   disabled: () => datatype.boolean(),
+  plugin: function () {
+    if (this.type == 'plugin') {
+      return {
+        id: `pl_${datatype.hexaDecimal(5)}`,
+        name: random.arrayElement(['azure', 'aws']),
```

Comment:
The current index of the instance is passed into the function, so be sure to add the parameter on line 14, then:
```suggestion
        name: i % 2 ? 'aws' : 'azure',
```